tests/RandomReadWrite.txt

```diff
@@ -7,7 +7,7 @@ writesPerTransactionA=0
 readsPerTransactionA=10
 writesPerTransactionB=10
 readsPerTransactionB=1
-; Transactions A/B
+; Transaction ratio: B/(A+B)
```

Comment:
This looks right, but since `A+B==1` it just reduces to `B`. Maybe a plain English description would be better, like "Fraction of transactions that will be of type B". Or, if we want to be more verbose, "Controls the ratio of A to B transactions by specifying the fraction of transactions that will be of type B".

Reply:
Well, I guess if you think of A and B here as the number of transactions of each type run rather than the fraction, `A+B!=1`. However, the description I proposed still works and is probably easier to reason about.

Reply:
Agree to your plain English description propose 👍